application-task-ui/src/main/resources/TaskManager/WebHomeSheet.xml

```diff
@@ -42,6 +42,14 @@
   {{missingLicenseMessage extensionName="taskmanager.extension.name"/}}
 #else
 
+{{html}}
```

Comment:
I dont think we should display this here. Rather, I think we sould have the `TaskTemplateList` page present in the admin section.